Apps/Sandcastle/gallery/3D Tiles Point Cloud.html

```diff
@@ -34,9 +34,7 @@
 }));
 
 tileset.readyPromise.then(function() {
-    var boundingSphere = tileset.boundingSphere;
-    viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0.0, -0.5, boundingSphere.radius));
-    viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
+    viewer.zoomTo(tileset, new Cesium.HeadingPitchRange(0.0, -0.5, tileset.boundingSphere.radius));
```

Comment:
Since this is used most commonly, can we make this the default? 

Then we can use the shorthand:

```
viewer.zoomTo(tileset).otherwise(function(error) {
    throw(error);
});
````